src/tests/utils/fixtures/order_fixture.py

```diff
@@ -2,12 +2,13 @@
 from src.constants.order_status import OrderStatus
 
 
-def build_order(order_id=None, status=None, assigned_chef_id=None):
+def build_order(order_id=None, status=None, assigned_chef_id=None, estimate_time=None):
 
     order = Order()
     order.id = order_id
     order.status = status or OrderStatus.NEW_ORDER
     order.assigned_chef_id = assigned_chef_id
+    order.estimated_time = estimate_time
```

Comment:
Why are you adding this here?

Reply:
Compute estimated time every time a order needs to evaluated.

Reply:
done.